scripts/servers.py.EXAMPLE

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
Pick one option (I personally am inclined towards 2)
1. make a README instead giving instructions on how to use this, and provide this as a sample in that file, so people can figure out how to make their own right away.
2. make the usernames and IP addresses command-line args of `bootstrap.py`
3. do both 1 & 2, using `len(argv)` or something to decide which to do

Reply:
`dbe-setup-and-run.py` (new name) takes a command line argument for a config file.
The setup of the config file is specified in a README.